docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -44,9 +45,8 @@ on:
     branches: ["master", "main"]
   schedule:
     - cron: '20 17 * * *' # Sets Semgrep to scan every day at 17:20 UTC.
-    
-permissions: 
-    security-events: write
+    # It is recommended to change the schedule to a random time.
```

Comment:
This comment really doesn't belong here - if you're going to retain it, it should be up with the cron line.